src/helpers/login.js

```diff
@@ -0,0 +1,45 @@
+export const iniciarSesion = (user, password) => {
+	let arreglo = [];
+	let validador = {
+		estado: true,
+		idUser: 0,
+	};
+
+	arreglo = JSON.parse(localStorage.getItem('usuarios'));
+
+	if (arreglo === null) {
+		validador.estado = true;
+	} else {
+		arreglo.map((usuarios) => {
+			if (usuarios.name === user && usuarios.password === password) {
+				validador.estado = false;
+				validador.idUser = arreglo.indexOf(usuarios);
+			}
+		});
+	}
+	return validador;
+};
+
+export const createUser = (name) => {
+	let arreglo = [];
+	let validador = {
+		estado: true,
+		idUser: 0,
+	};
+
+	arreglo = JSON.parse(localStorage.getItem('usuarios'));
```

Comment:
arreglo should be defined as a const and avoud define on line 24 as an empty array